test/functional/pdo_sqlsrv/MsCommon.inc

```diff
@@ -447,155 +444,6 @@ function is_col_enc()
-            [NCharCol] [nchar](10) NULL,
-            [NTextCol] [ntext] NULL,
-            [NumCol] [numeric](18, 0) NULL,
-            [NVarCharCol] [nvarchar](50) NULL,
-            [NVarCharMaxCol] [nvarchar](max) NULL,
-            [RealCol] [real] NULL,
-            [SmallDTCol] [smalldatetime] NULL,
-            [SmallIntCol] [smallint] NULL,
-            [SmallMoneyCol] [smallmoney] NULL,
-            [TextCol] [text] NULL,
-            [TimeCol] [time](7) NULL,
-            [TinyIntCol] [tinyint] NULL,
-            [Guidcol] [uniqueidentifier] NULL,
-            [VarbinaryCol] [varbinary](50) NULL,
-            [VarbinaryMaxCol] [varbinary](max) NULL,
-            [VarcharCol] [varchar](50) NULL,
-            [VarcharMaxCol] [varchar](max) NULL,
-            [XmlCol] [xml] NULL
-        ) ON [PRIMARY] TEXTIMAGE_ON [PRIMARY]
-        
-        ";
-          
-        $stmt = $conn->query( $create_query );
-        
-        for ($i =0; $i<= 0 ; ++ $i)
-        {
-          $insert_query = 
-                    
-                  " INSERT INTO " .$table2 . " VALUES (".
-                    $int_col[$i] . $comma . 
-                    $binary_col[$i] . $comma .
-                    "0" . $comma . 
-                    $string_col[$i] . $comma . 
-                    "Convert(date, ". $date_col[$i] . $closing_brace . 
-                    "Convert(datetime, ". $date_col[$i] . $closing_brace .
-                    "Convert(datetime2(7),". $date_col[$i] . $closing_brace . 
-                    "Convert(datetimeoffset(7)," . $date_col[$i] . $closing_brace . 
-                    $decimal_col[$i] . $comma . 
-                    $decimal_col[$i] .$comma . 
-                    $binary_col[$i] . $comma . 
-                    $int_col[$i] . $comma .
-                    $decimal_col[$i] . $comma . 
-                    $string_col[$i]. $comma . 
-                    $large_string_col[$i]. $comma. 
-                    $int_col[$i]. $comma . 
-                    $string_col[$i]. $comma . 
-                    $large_string_col[$i]. $comma .
-                    $decimal_col[$i]. $comma . 
-                    "Convert(smalldatetime, ". $date_col[$i]. $closing_brace . 
-                    $int_col[$i]. $comma . 
-                    $decimal_col[$i]. $comma . 
-                    $large_string_col[$i]. $comma . 
-                    "Convert(time(7), ". $date_col[$i] . $closing_brace . 
-                    $int_col[$i] . $comma . 
-                    $guid_col[$i] . $comma . 
-                    $binary_col[$i] . $comma . 
-                    $binary_col[$i] . $comma . 
-                    $string_col[$i] . $comma . 
-                    $large_string_col[$i] . $comma .
-                    $xml_col[$i] . 
-                    ")";
-         
-          $stmt = $conn->query ($insert_query );   
-        }
-        
-  }
-  catch(Exception $e)
-  {
-     var_dump( $e);
-     exit;
-  }
-}
-
 function teardown()
 {
     // TBD
```

Comment:
There are quite a few methods in MsCommon.inc that are probably not very useful. Next time please consider merging your changes in mid-refactor to this one and removing the unused helper methods.